overlord/snapstate/autorefresh.go

```diff
@@ -119,7 +120,7 @@ func (m *autoRefresh) Ensure() error {
 	if !m.nextRefresh.IsZero() {
 		if m.lastRefreshSchedule != refreshScheduleStr {
 			// the refresh schedule has changed
-			logger.Debugf("Option refresh.schedule changed.")
+			logger.Debugf("Option refresh.timer changed.")
```

Comment:
(nitpick) is this so? I mean, do we know if refresh.schedule or refresh.timer changed here? Not that it matters, just wondering.